Source/WebCore/rendering/RenderCombineText.cpp

```diff
@@ -99,7 +100,7 @@ void RenderCombineText::combineTextIfNeeded()
 
     // An ancestor element may trigger us to lay out again, even when we're already combined.
     if (m_isCombined)
-        RenderText::setRenderedText(originalText());
+        RenderText::setRenderedText(m_renderedText);
```

Comment:
Seems like the original intended was to copy the string.

@darinadler @cdumez The original Blink patch did `RenderText::setRenderedText(m_renderedText.impl())`, although I'm not sure if that's the common pattern.

Reply:
This looks right to me. No return to return the StringImpl here.

Reply:
Seems like maybe the Blink merge simply isn't working, the initial version without any of my requested changes had the test failing on Windows.